apps/web/src/components/EmployeeSearchInput/EmployeeSearchInput.stories.tsx

```diff
@@ -82,6 +82,21 @@ export const NoResult: Story = {
   },
 };
 
+export const Required: Story = {
```

Comment:
The [Chromatic snapshot](https://www.chromatic.com/component?appId=61e099a1bb1465003a73d3ce&csfId=components-employeesearchinput&specName=Required&buildNumber=10617&k=67dc0d957aad52dba614f403-1200px-snapshot-true&h=4&b=-2) doesn't capture the state with the error. I don't think we have used it yet, but would the [play function](https://storybook.js.org/docs/writing-stories/play-function#writing-stories-with-the-play-function) make sense for this one?

Reply:
Great suggestion. Should be better now after [submit form for error display](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13053/commits/bd82436ab737a78f63f0d918b152ecef50c22a01)